jetty-core/jetty-osgi/src/main/java/org/eclipse/jetty/osgi/OSGiDeployableBundleMetadata.java

```diff
@@ -35,27 +30,47 @@
 import org.osgi.framework.Bundle;
 import org.osgi.framework.FrameworkUtil;
 import org.osgi.framework.ServiceRegistration;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.osgi.framework.Version;
 
 /**
- * OSGiApp
- *
- * Base class representing info about a WebAppContext/ContextHandler to be deployed into jetty.
+ * Metadata useful for a deployment that will result in a {@link org.eclipse.jetty.server.handler.ContextHandler}
  */
-public class OSGiApp extends App
+public class OSGiDeployableBundleMetadata
```

Comment:
Classname doesn't need OSGi prefix - I only used that for those classes that are implementing deployment manager classes to distinguish them. This can be just `DeployableBundleMetadata`, or even just `BundleMetadata`.

Reply:
ok